### ISSUE TITLE
The Kind Keptn Tries to Bind to High Port and Fails

### ISSUE THREAD

**Issue by agardnerIT** [keptn/keptn#7112]
```
The connection to the server 0.0.0.0:40813 was refused - did you specify the right host or port?
```

For some users, thekindkeptn attempts to bind to a high port and fails. Therefore kubectl never authenticates and thus everything else fails.

The line going wrong in the install_script.sh is: https://github.com/agardnerIT/thekindkeptn/blob/e870cb97b23456d32e99f07e2831673891383d02/install_script.sh#L36
`k3d cluster create mykeptn --config=/root/k3dconfig.yaml --wait`

Where the k3dconfig.yaml is:
```
---
apiVersion: k3d.io/v1alpha3
kind: Simple
name: mykeptn
servers: 1
#agents: 1
ports:
  - port: 80:80
    nodeFilters:
    - loadbalancer
options:
  k3d: # k3d runtime settings
    wait: true # wait for cluster to be usable before returning; same as `--wait` (default: true)
    timeout: "60s" # wait timeout before aborting; same as `--timeout 60s`
    #disableLoadbalancer: true
  k3s: # options passed on to K3s itself
    extraArgs: # additional arguments passed to the `k3s server` command
      - arg: --disable=traefik
        nodeFilters:
          - server:*
  kubeconfig:
    updateDefaultKubeconfig: true # add new cluster to your default Kubeconfig; same as `--kubeconfig-update-default` (default: true)
    switchCurrentContext: true # also set current-context to the new cluster's context; same as `--kubeconfig-switch-context` (default: true)
```

**Comment by agardnerIT** [comment]
Log output of broken install:

```
-- Installing Versions --
Keptn: 0.13.2
Job Executor Service: 0.1.7
-- Bringing up a cluster --
INFO[0000] Using config file /root/k3dconfig.yaml (k3d.io/v1alpha3#simple) 
WARN[0000] Default config apiVersion is 'k3d.io/v1alpha4', but you're using 'k3d.io/v1alpha3': consider migrating. 
INFO[0000] portmapping '80:80' targets the loadbalancer: defaulting to [servers:*:proxy agents:*:proxy] 
INFO[0000] Prep: Network                                
INFO[0000] Created network 'k3d-mykeptn'                
INFO[0000] Created image volume k3d-mykeptn-images      
INFO[0000] Starting new tools node...                   
INFO[0000] Starting Node 'k3d-mykeptn-tools'            
INFO[0001] Creating node 'k3d-mykeptn-server-0'         
INFO[0001] Creating LoadBalancer 'k3d-mykeptn-serverlb' 
INFO[0001] Using the k3d-tools node to gather environment information 
INFO[0001] HostIP: using network gateway 172.19.0.1 address 
INFO[0001] Starting cluster 'mykeptn'                   
INFO[0001] Starting servers...                          
INFO[0001] Starting Node 'k3d-mykeptn-server-0'         
INFO[0014] All agents already running.                  
INFO[0014] Starting helpers...                          
INFO[0014] Starting Node 'k3d-mykeptn-serverlb'         
INFO[0022] Injecting records for hostAliases (incl. host.k3d.internal) and for 2 network members into CoreDNS configmap... 
INFO[0024] Cluster 'mykeptn' created successfully!      
INFO[0025] You can now use it like this:                
kubectl cluster-info
-- Waiting for all resources to be ready (timeout 2 mins) --
The connection to the server 0.0.0.0:40813 was refused - did you specify the right host or port?
```

Equivalent log output of working installation:

```
-- Installing Versions --
Keptn: 0.13.2
Job Executor Service: 0.1.7
-- Bringing up a cluster --
INFO[0000] Using config file /root/k3dconfig.yaml (k3d.io/v1alpha3#simple)
WARN[0000] Default config apiVersion is 'k3d.io/v1alpha4', but you're using 'k3d.io/v1alpha3': consider migrating.
INFO[0000] portmapping '80:80' targets the loadbalancer: defaulting to [servers:*:proxy agents:*:proxy]
INFO[0000] Prep: Network
INFO[0000] Created network 'k3d-mykeptn'
INFO[0000] Created image volume k3d-mykeptn-images
INFO[0000] Starting new tools node...
INFO[0000] Starting Node 'k3d-mykeptn-tools'
INFO[0001] Creating node 'k3d-mykeptn-server-0'
INFO[0001] Creating LoadBalancer 'k3d-mykeptn-serverlb'
INFO[0001] Using the k3d-tools node to gather environment information
INFO[0002] Starting cluster 'mykeptn'
INFO[0002] Starting servers...
INFO[0002] Starting Node 'k3d-mykeptn-server-0'
INFO[0007] All agents already running.
INFO[0007] Starting helpers...
INFO[0007] Starting Node 'k3d-mykeptn-serverlb'
INFO[0014] Injecting records for hostAliases (incl. host.k3d.internal) and for 2 network members into CoreDNS configmap...
INFO[0016] Cluster 'mykeptn' created successfully!
INFO[0016] You can now use it like this:
kubectl cluster-info
-- Waiting for all resources to be ready (timeout 2 mins) --
pod/coredns-96cc4f57d-cmcw2 condition met
...
```

Only difference I see is, in the broken one:

```
INFO[0001] Using the k3d-tools node to gather environment information 
INFO[0001] HostIP: using network gateway 172.19.0.1 address 
```

**Comment by agardnerIT** [comment]
https://github.com/k3d-io/k3d/issues/1008

**Comment by agardnerIT** [comment]
Solved with `thekindkeptn:0.0.9` Thanks to @dixon1e for assisting in the investigation :)
